web-app/js/portal/filter/BaseFilterPanel.js

```diff
@@ -126,7 +126,8 @@ Portal.filter.BaseFilterPanel.newFilterPanelFor = function(cfg) {
     }
     else {
         //Filter hasn't been defined
-        log.error("Could not create filter panel for type'" + type + "'");
+        log.error("Could not find correct filter panel class for type'" + type + "'");
```

Comment:
But maybe a more succinct error message can be used here, such as "Unknown filter type 'TYPE'". Perhaps also add the layer name if you can - will assist in fixing the problem.